interpreter/python/python.go

```diff
@@ -655,33 +656,39 @@ func (d *pythonData) readIntrospectionData(ef *pfelf.File, symbol libpf.SymbolNa
 
 // decodeStub will resolve a given symbol, extract the code for it, and analyze
 // the code to resolve specified argument parameter to the first jump/call.
-func decodeStub(ef *pfelf.File, addrBase libpf.SymbolValue, symbolName libpf.SymbolName,
-	argNumber uint8) libpf.SymbolValue {
-	symbolValue, err := ef.LookupSymbolAddress(symbolName)
+func decodeStub(
+	ef *pfelf.File,
+	memoryBase libpf.SymbolValue,
+	symbolName libpf.SymbolName,
+) (libpf.SymbolValue, error) {
+	codeAddress, err := ef.LookupSymbolAddress(symbolName)
 	if err != nil {
-		return libpf.SymbolValueInvalid
+		return libpf.SymbolValueInvalid, fmt.Errorf("lookup %s failed: %w",
```

Comment:
Does `LookupSymbolAddress()` return an error that is exposed and can be handled in some way? Otherwise we should use `%v` over `%w`:
```suggestion
		return libpf.SymbolValueInvalid, fmt.Errorf("lookup %s failed: %v",
```